test/util/testfactory/txs.go

```diff
@@ -31,3 +32,27 @@ func GenerateRandomTxs(count, size int) types.Txs {
 	}
 	return txs
 }
+
+func GetRandomSlice(size int) (start int, length int) {
+	length = rand.Intn(size + 1)
+	start = rand.Intn(size - length + 1)
+	return
```

Comment:
[uber nit] this codebase strives to avoid [naked returns](https://go.dev/tour/basics/7) because for large function bodies they can be less readable than non-naked returns. This isn't a large function body but proposal to use non-naked return for readability:

```suggestion
	return start, length
```

Maintainers should consider enabling a lint rule that complains if naked returns are used.